go/vt/vtgate/querylogz.go

```diff
@@ -45,12 +45,15 @@ var (
 				<th>Duration</th>
 				<th>Plan Time</th>
 				<th>Execute Time</th>
+				<th>Mirror Source Execute Time</th>
```

Comment:
Is `Execute Time` and `Mirror Source Execute Time` same or different? Ideally a Mirror execution should not impact the query execution time.

Reply:
they are measured in different places. `Execute Time` is measured here:

https://github.com/vitessio/vitess/blob/0bf4276e320c2f42f6d2d21adc85b692f4db29c2/go/vt/vtgate/executor.go#L383

`Mirror Source Execute Time` is measure here:

https://github.com/vitessio/vitess/blob/0bf4276e320c2f42f6d2d21adc85b692f4db29c2/go/vt/vtgate/engine/mirror.go#L107

so `Mirror Source Execute Time` should include but may be slightly less than `Execute Time`. The reason we have `Mirror Source Execute Time` is to have an apples-to-apples way to compare the performance of mirroring.